lib/rex/proto/mssql/client_mixin.rb

```diff
@@ -206,6 +206,15 @@ def mssql_parse_tds_reply(data, info)
       when 50
         col[:id] = :bit
 
+      when 99
+        col[:id] = :ntext
+        col[:max_size] = data.slice!(0, 4).unpack('V')[0]
+        col[:codepage] = data.slice!(0, 2).unpack('v')[0]
+        col[:cflags] = data.slice!(0, 2).unpack('v')[0]
+        col[:charset_id] =  data.slice!(0, 1).unpack('C')[0]
+        col[:namelen] = data.slice!(0, 1).unpack('C')[0]
+        col[:table_name] = data.slice!(0, (col[:namelen] * 2) + 1).gsub("\x00", '')
```

Comment:
I think there's a directive for null terminated strings so this `gsub` isn't required 👀 

> https://apidock.com/ruby/String/unpack
> Z            | [String](https://apidock.com/ruby/String)  | null-terminated string

Reply:
unless this is needed for utf16 support?

Reply:
I think it is, but I'm not positive. Do you have any suggestions for verifying this?